packages/core/content-releases/admin/src/components/CMReleasesContainer.tsx

```diff
@@ -208,13 +209,24 @@ export const CMReleasesContainer = () => {
   } = useCMEditViewDataManager();
   const params = useParams<{ id?: string }>();
 
-  const toggleAddActionToReleaseModal = () => setShowModal((prev) => !prev);
+  const canFetch = Boolean(params.id && contentType?.uid);
```

Comment:
If instead of Boolean you check if they're `!= null`, do you still have to do the TS asserting below?